src/components/e-date.vue

```diff
@@ -297,21 +290,17 @@
 
       grid-area: input;
       width: 14ch; // 8 letters + 2 dividers + some, because IE11 does not count outer spacing of 0
```

Comment:
todo: please use the value that was previously inside the `@supports` block and was removed with one of your latest commits.

```suggestion
      width: var(--e-date-format-length, 10ch); // 8 letters + 2 dividers
```